### ISSUE TITLE
[BUG] Wrong light level on ground near players feet when freelooking

### ISSUE THREAD

**Issue by electricbrass** [odamex/odamex#1061]
**Describe the bug**
When looking down, in the bottom half-ish of the screen, the floor is too bright in dark areas.

**Build that the bug occurred in**
`stable` commit [3b3ed95](https://github.com/odamex/odamex/commit/3b3ed954e71ad4cc1f895f8b2075b8cae4db24dc)

**To Reproduce**
Go into a dark area and look down.

**Expected behavior**
No sudden change in the floor's light level.

**Screenshots, NetDemos, & Crash Dumps**
https://github.com/user-attachments/assets/47a9dfad-672e-4a09-a5fa-c5f931554347

**Comment by rakohus** [comment]
This might be correct behavior, even if it looks dumb. It treats all floors within that range (and ceilings too i'd guess) this way in both 8bpp and 32bpp. I added a staircase that increases by 4 units each up to the player's height behind the spawn in that same map to demonstrate. Attached are videos of Oda in both 8/32bpp, and a screenshot of chocolate doom showing the same:

https://github.com/user-attachments/assets/b7ef1a6d-d199-4198-ab24-a1f6f2c0192a

https://github.com/user-attachments/assets/17bbc947-394e-45b1-a9d7-a8a3f50af29e

![image](https://github.com/user-attachments/assets/98d59b2c-665a-4189-8a42-2a76b22c082a)

**Comment by electricbrass** [comment]
Should this be closed as not a bug?

**Comment by rakohus** [comment]
Yeah, I think so. I'll close it, but I guess we could revisit it later as part of some other greater renderer improvement thing. Maybe. Who knows
